### PR TITLE
jenkins: Allow HTTP Instances

### PR DESCRIPTION
**Pull request by vkhitrin**
## Description

Allow the extension to connect to HTTP instances.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @vkhitrin 🔥 

Do you mind checking this @k8scat

**Comment by pernielsentikaer** [comment]
@k8scat are you still around?

**Comment by k8scat** [comment]
I'll have a look later today

**Comment by pernielsentikaer** [comment]
Do you have time to check it @k8scat 🙂

**Comment by pernielsentikaer** [comment]
@k8scat are you still able to check?

File: extensions/jenkins/src/lib/api.ts [hunk 1]
```diff
@@ -105,9 +106,15 @@ export class JenkinsAPI {
   }
 
   async request(url: RequestInfo, init?: RequestInit) {
-    const httpsAgent = new https.Agent({
-      rejectUnauthorized: !this.jenkins.unsafeHttps,
-    });
+    let urlAgent;
```
Comment (k8scat) [inline extensions/jenkins/src/lib/api.ts:109]: The variable is not assigned a value

Comment (vkhitrin) [reply extensions/jenkins/src/lib/api.ts:109]: Oh wow, thanks for catching it 😓.
I will fix this!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**